File: tests/sample.cs
// sample.cs
using System;
using System.Collections.Generic;

public record User(int Id, string Name);

public class Program {
    public static int Add(int a, int b) => a + b;

    public static void Main() {
        var users = new List<User> { new(1, "Ada"), new(2, "Linus") };
        Console.WriteLine(users[0]);
        Console.WriteLine(Add(2, 3));
    }
}

File: tests/sample.cpp
// sample.cpp
#include <iostream>
#include <vector>

struct User {
    int id;
    std::string name;
};

int add(int a, int b) {
    return a + b;
}

int main() {
    std::vector<User> users{{1, "Ada"}, {2, "Linus"}};
    std::cout << users[0].name << "\n";
    std::cout << add(2, 3) << "\n";
    return 0;
}
